### DISCUSSION TITLE
Trouver le mail de quelqu'un facilement

### DISCUSSION THREAD

**Discussion by astranchet** [betagouv/cap#2] [Besoin]
Je connais la page communauté, mais ça ne me dit pas comment contacter la personne. Parfois, je ne la trouve pas sur Mattermost, et elle n'a pas d'email beta (mais un champ en @autrechose.gouv.fr). Bref, j'aimerai bien savoir comment la contacter facilement

**Comment by revolunet** [comment]
ce serait une info à rajouter dans la fiche du membre ?

**Reply by astranchet** [reply]
En l'état des choses, je ne sais pas si on devrait le mettre dans la fiche, alors qu'on a déjà l'info dans une autre base côté espace membre (email primaire, email secondaire, etc)

**Reply by revolunet** [reply]
deux choix :
 - on l'extrait l'email ou handle mattermost de l'espace membre pour le mettre sur la fiche
 - on fait un lien depuis la fiche publique vers une fiche privée dans l'espace ~~communauté~~ membre
 
 perso je vote pour le 1

**Reply by astranchet** [reply]
On en parle à l'oral ? Je suis embrouillée quand tu parles de fiche, c'est le fichier .md de github ? ou autre chose ?

**Reply by gauthierna** [reply]
@astranchet est-ce que la piste de solution que je mentionne ici (https://github.com/betagouv/cap/discussions/13) pourrait te convenir ?

**Reply by astranchet** [reply]
@gauthierna j'aime beaucoup la solution #13 mais c'est 2 problèmes différents : 

Quand tu parles à quelqu'un sur MM, effectivement, on a pas d'infos sur l'équipe et tout ça (par contre, on a le mail ! et ça je m'en sers souvent !).
Mais dans ce ticket, je parle plutôt du moment où quand on lit une fiche équipe, on a bien les noms des membres mais... pas de moyens de les contacter.

**Comment by revolunet** [comment]
toujours d'actu ?
